examples/tutorials/colabs/material_tutorial.ipynb

```diff
@@ -0,0 +1,261 @@
+{
```

Comment:
I guess there is no easy way to test this Colab until this PR merges and my C++ changes get into the Hab-sim conda build.